CMakeLists.txt

```diff
@@ -84,17 +86,16 @@ add_dependencies(jsonfortran json-fortran)
 set_property(TARGET jsonfortran PROPERTY IMPORTED_LOCATION ${BINARY_DIR}/lib/libjsonfortran.a)
 
 # oasis external project
-# json-fortran external project
 ExternalProject_Add(oasis3-mct
```

Comment:
We're still using CMake to build and link to `oasis3-mct`. Again I think we want to use build with spack and use pkgconfig to find paths necessary for linking.

Reply:
This PR is not related to using Spack.